doc: profiling remote compute nodes (#7426)
risingwavelabs/risingwave: docs/cpu-profiling.md

```shell
nperf flamegraph perf.data > perf.svg
```

## Profiling remote compute nodes
You can profile remote compute nodes from a local machine by simply type the following command.
```shell
./risedev ctl profile --sleep [seconds]
```
All compute nodes will be profile for a given `seconds` time and generated flame graph will be transferred to your local machine.

> Note: To profile our compute nodes remotely, please make sure all remote nodes have a public ip address accessible from your local machine (The one)

Reviewer: ```suggestion
Note: To profile compute nodes remotely, please make sure all remote nodes have a public IP address accessible from your local machine (The one 😴?)
```

> Note: To profile compute nodes remotely, please make sure all remote nodes have a public IP address accessible from your local machine (The one you are running `./risedev`)

Reviewer: ```suggestion
Note: To profile compute nodes remotely, please make sure all remote nodes have a public IP address accessible from your local machine (where you are running `risedev`).
```
